test/mox_test.exs

```diff
@@ -190,4 +190,30 @@ defmodule MoxTest do
       end
     end
   end
+
+  describe "allow/2" do
+    test "allows different processes to share mocks" do
+      test_pid = self()
+
+      child_pid = spawn fn ->
+        receive do
+          :call_mock ->
+            CalcMock.add(1, 1)
+            CalcMock.mult(1, 1)
+            send(test_pid, :verify)
```

Comment:
Is it helpful to verify that this process gets the correct return values or do you feel like that's covered inherently by the call to verify?

```
child_pid = spawn fn ->
  recieve do
    :call_mock ->
      add_result = CalcMock.add(1, 1)
      mult_result = CalcMock.mult(1, 1)
      send(test_pid, {:verify, add_result, mult_result})
  end
end
...
receive do
  {:verify, :expected, :stubbed} -> verify!()
end
```

Reply:
I have added this, also added a timeout/after clause to receive so the tests won't block infinitely if something breaks